setup.py

```diff
@@ -46,6 +47,20 @@
 
 LIBDDWAF_VERSION = "1.6.0"
 
+CHECKSUMS = {
```

Comment:
```suggestion
LIBDDWAF_CHECKSUMS = {
```

and same for `verify_checksum` -> `verify_libddwaf_checkum` ?

this is a nit, not sure it actually matters. feel free to resolve without making any changes.